python/src/fsoi/web/batch_wrapper.py

```diff
@@ -346,16 +346,73 @@ def create_plots(request, center, objects):
         cycle_ints.append(int(c))
 
     # create the plots
-    platform = loi.Platforms('OnePlatform')
     for qty in ['TotImp', 'ImpPerOb', 'FracBenObs', 'FracNeuObs', 'FracImp', 'ObCnt']:
         try:
             plot_options = loi.getPlotOpt(qty, cycle=cycle_ints, center=center,
-                                          savefigure=True, platform=platform, domain='Global')
+                                          savefigure=True, platform=loi.Platforms('OnePlatform'), domain='Global')
             plot_options['figname'] = '%s/plots/summary/%s/%s_%s_%s' % \
                                       (request['root_dir'], center, center, qty, cycle_id)
             loi.summaryplot(df, qty=qty, plotOpt=plot_options, std=df_std)
         except Exception as e:
-            print(e)
+            log.error('Failed to generate plots for %s' % qty, e)
+
+
+def aggregate_by_platform(df):
+    """
+    Aggregate all of the data by platform using the unified platform list (e.g. [MODIS_Wind and
+    AMV-MODIS] go under a single index called MODIS Wind).
+    :param df: The original data frame, which will be deleted upon successful completion
+    :return: {pandas.DataFrame} A new data frame with data aggregated by unified platform list
+    """
+    # turn the unified platform list inside-out for quick look up
+    map = {}
```

Comment:
One more comment. Suggest renaming `map`.  This overloads a library function name.